### ISSUE TITLE
Org Telemetry can throw timeout error for count of Recurring Donation records in larger orgs

### ISSUE THREAD

**Issue by judisohn** [SalesforceFoundation/Cumulus#3676]
[Thread](https://powerofus.force.com/0D58000004Xpmpl) (Partner group).

Customer is getting error notification:

```
Message: First error: [REQUEST_RUNNING_TOO_LONG] Your request was running for too long, 
and has been stopped.
Context: npsp__UTIL_OrgTelemetry_BATCH
```

This is related to counting Recurring Donation records.

As this batch only runs at the time of the install/upgrade of NPSP, the error can be ignored until it's fixed.

**Comment by judisohn** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000007fOKVAA2" target="_blank">W-028851</a>

**Comment by mrbelvedere** [comment]
Included in beta release 3.142 (Beta 7)
